<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SideBarWidget" name="side_bar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2000</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="analysis_container">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>500</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <widget class="QWidget" name="analysis_containerPage1"/>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="ArrowButton" name="back_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ArrowButton" name="forward_button">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="new_action"/>
    <addaction name="resume_action"/>
    <addaction name="save_action"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="resume_action">
   <property name="text">
    <string>Resume evaluation</string>
   </property>
   <property name="toolTip">
    <string>Resume previous evaluation</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
  </action>
  <action name="new_action">
   <property name="text">
    <string>New evaluation</string>
   </property>
   <property name="toolTip">
    <string>Start a new evaluation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Save evaluation</string>
   </property>
   <property name="toolTip">
    <string>Save current evaluation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SideBarWidget</class>
   <extends>QWidget</extends>
   <header>tipeval.ui.widgets.side_bar_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowButton</class>
   <extends>QPushButton</extends>
   <header>tipeval.ui.widgets.arrow_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
